src/main/java/org/generationcp/middleware/service/api/study/StudyEntryDto.java

```diff
@@ -34,6 +34,8 @@ public class StudyEntryDto implements Serializable {
 
 	private String guid;
 
+	private String breedingMethodId;
```

Comment:
@cheryleigh if we use abbreviation for implementing the popup we dont need to add this attribute, right?